packages/page-template-modal/src/styles/page-template-modal.scss

```diff
@@ -217,234 +163,140 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 
 .template-selector-item__media {
 	width: 100%;
+	height: 100%;
 	display: block;
 	position: absolute;
 	top: 0;
 	left: 0;
+	object-fit: cover;
 }
 
-.page-template-modal__form {
-	@media screen and ( min-width: $breakpoint-mobile ) {
-		max-width: 20%;
-	}
-
-	@media screen and ( min-width: $breakpoint-tablet ) {
-		max-width: 30%;
-	}
-}
-
-.page-template-modal__form-title {
-	font-weight: bold;
-	margin-bottom: 1em;
-	text-align: center;
+.page-template-modal__sidebar {
 	@media screen and ( min-width: $breakpoint-mobile ) {
-		text-align: left;
-	}
-}
-
-.page-template-modal__buttons {
-	position: absolute;
-	right: 0;
-	top: 0;
-	z-index: 10;
-	height: 56px;
-	display: flex;
-	align-items: center;
-	padding-right: 24px;
+		position: fixed;
+		width: calc( #{$sidebar-width} - 24px );
+		pointer-events: none;
```

Comment:
i don't understand the need to remove pointer events and then add it back on the children, is this still neccessary?

Reply:
Ah yeah, I should have had a comment here. This is why I did it, but let me know if you think if the behaviour isn't worth the added complexity.

I want the scrollbar for the layout list to behave as if it's the scrollbar for the whole modal (rather than a child scrollbar that's just for one of the columns inside the modal). This is mostly achieved by giving the category sidebar a `position: fixed`.
However if you use the mouse wheel while hovered over the sidebar it won't scroll the modal's scrollbar, which is the UX I was looking for. So I'm setting all the pointer events manually since the default browser behaviour wasn't giving me what I wanted. Disabling pointer events on the sidebar will allow the scroll wheel to work while moused over the sidebar, and then I need to re-enable pointer events again so that the clicks actually go to the buttons.

I'd recommend commenting out the `pointer-events` locally to get the idea of what I'm talking about.

Reply:
ahhh ok, TIL about this scrolling behaviour edge case and workaround 👍 yea it would probably be worth adding a comment to the code about that. Nice work being able to address it!